content/docs/02-getting-started/00-alpha.mdx

```diff
@@ -0,0 +1,212 @@
+---
+title: AI SDK 5 Alpha
+description: Get started with the latest Alpha version of the AI SDK.
+---
+
+# AI SDK 5 - Alpha
+
+<Note>
+  This is an early preview — AI SDK 5 is under active development. APIs may
+  change without notice. Pin production to v4 (ai@^4).
+</Note>
+
+## Installation
+
+To install the AI SDK 5 - Alpha, run the following command:
+
+```bash
+# replace with your provider and framework
+npm install ai@canary @ai-sdk/[your-provider]@canary @ai-sdk/[your-framework]@canary
+```
+
+## What's new in AI SDK 5?
+
+AI SDK 5 represents a full redesign of the AI SDK's protocol and architecture, building a more robust platform that enables quick and reliable support for all new features from AI providers as they emerge.
+
+### Why AI SDK 5?
+
+When we originally designed the v1 protocol over a year ago, the standard interaction pattern with language models was simple: text in, text or tool call out. But today's LLMs go way beyond text and tool calls, generating reasoning, sources, images and more. Additionally, new use-cases like computer using agents introduce a fundamentally new approach to interacting with language models that made it near-impossible to support in a unified approach with our original architecture.
+
+We needed a protocol designed for this new reality. While this is a breaking change that we don't take lightly, it's provided an opportunity to rebuild the foundation and add powerful new features.
+
+While we've designed AI SDK 5 to be a substantial improvement over previous versions, we're still in active development. You might encounter bugs or unexpected behavior. We'd greatly appreciate your feedback and bug reports—they're essential to making this release better. Please share your experiences and suggestions with us through GitHub issues or GitHub discussions.
+
+### Feature Overview
+
+- **V2 Protocol** - redesigned from the ground up for modern AI capabilities
+- **Data Parts** - stream type-safe arbitrary data parts to the client
```

Comment:
would start with ui message vs model message, and then fold things like stream writer, data parts, metadata under ui messages